spa-dashboard-angular/src/app/room-detail/room-detail.component.html

```diff
@@ -0,0 +1,31 @@
+<div *ngIf="room">
+
+  <h2>Room Details</h2>
+
+  <dl class="property-list">
+    <dt class="property-list__key">Room name</dt>
+    <dd class="property-list__value">{{room.name}}</dd>
+  </dl>
+
+  <dl class="property-list">
+    <dt class="property-list__key">Room reference</dt>
+    <dd class="property-list__value"><div class="code">{{room.roomRef}}</div></dd>
+  </dl>
+
+  <dl class="property-list">
+    <dt class="property-list__key">Max places</dt>
+    <dd class="property-list__value">{{room.maxPlaces}}</dd>
+  </dl>
+
+  <dl class="property-list">
+    <dt class="property-list__key">Has remote equipment:</dt>
+    <dd class="property-list__value,code"><div class="code">{{room.hasRemoteEquipment}}</div></dd>
+  </dl>
```

Comment:
```suggestion
  <dl class="property-list">
    <dt class="property-list__key">Room name</dt>
    <dd class="property-list__value">{{room.name}}</dd>

    <dt class="property-list__key">Room reference</dt>
    <dd class="property-list__value"><div class="code">{{room.roomRef}}</div></dd>

    <dt class="property-list__key">Max places</dt>
    <dd class="property-list__value">{{room.maxPlaces}}</dd>

    <dt class="property-list__key">Has remote equipment:</dt>
    <dd class="property-list__value,code"><div class="code">{{room.hasRemoteEquipment}}</div></dd>
  </dl>
```

Similar to the other detail component.